src/components/DpDataTable/DpTableRow.vue

```diff
@@ -79,7 +79,9 @@
       :class="{ 'is-open': expanded }"
       :title="Translator.trans(expanded ? 'aria.collapse' : 'aria.expand')"
       @click="toggleExpand(item[trackBy])">
-      <dp-wrap-trigger :expanded="expanded" />
+      <dp-wrap-trigger
+          :data-cy="`isExpandableWrapTrigger:${$attrs.index}`"
+          :expanded="expanded" />
```

Comment:
please indent with 2 blank spaces, not with 4. For all changes....
And please configure Your IDE to do so by default for vue and js files

Reply:
https://github.com/demos-europe/demosplan-ui/pull/812/commits/529a27ec30bc0ff1f5bd4f5e2c2285c3c74fee4b